### ISSUE TITLE
REST API tests are not passing correctly on oardocker/Jessie 2.5 branch

### ISSUE THREAD

**Issue by bzizou** [oar-team/oar#78]
Seems that there are some changes about returned links.
For example:

  1) OarApi JOBS CHECKINGS: /jobs data structure (using state=Running,Launching,Waiting&limit=2) should return a correct self link
     Failure/Error: @api.get_self_link_href.should == "#{APIPATH}jobs?state=Running%2CLaunching%2CWaiting&limit=2&offset=0"
      **expected: "/oarapi-priv/jobs?state=Running%2CLaunching%2CWaiting&limit=2&offset=0"**
          **got: "/oarapi-priv/jobs/jobs?state=Running%2CLaunching%2CWaiting&limit=2&offset=0"**(using ==)
     # ./format_checking_spec.rb:145:in `block (4 levels) in <top (required)>'

  17) OarApi JOB INFOS CHECKING: /jobs/<id> data structure (with finished job) it should behave like Job resources link should be correct
     Failure/Error: @api.get_link_href('resources').should == "#{APIPATH}jobs/#{@api.value['id']}/resources"
       **expected: "/oarapi-priv/jobs/1/resources"**
           **got: "/oarapi-priv/jobs/1/jobs/1/resources" (using ==)**
     Shared Example Group: "Job" called from ./format_checking_spec.rb:298
     # ./shared_examples.rb:48:in `block (2 levels) in <top (required)>'

**Comment by npf** [comment]
Should we delay 2.5.6 ?

**Comment by bzizou** [comment]
We should first check if tests are passing on oar-vagrant. There's no reason to delay if the tests are passing with vagrant and it should only be a oar-docker issue.

**Comment by npf** [comment]
2.5.6+rc4 is ready for test:
~~~
cd oar-vagrant/deb_debian7_pgsql
export OAR_FTP_DISTRIB=sid_beta
. ../misc/env.sh
vagrant up
~~~

**Comment by bzizou** [comment]
Tests are passing on oar-vagrant.
I updated the wiki page of the api tests to take into account the new port of the API (6668): https://oar.imag.fr/wiki:running_the_api_tests#running_the_tests
I'll check oar-docker, probably to re-enable the apache configuration (links problems of the api are probably due to nginx)

**Comment by bzizou** [comment]
Fixed by commits 176ea98c016d01f923d9b5cdcd067f0913efbd40 and 13413212fcbe90a149f1c104e681e6e68e367bf4
